project-base/scripts/install.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 set -e
 
-echo "Start with specifying your operating system: \
```

Comment:
Initially, I removed it because I thought it's forgotten. But when I tested it now, I realized the backslash removes the next blank line.
I think this type of writing is confusing. So I suggest to either remove backslash together with next blank line or keep blank line as it is.
I prefer second variant, but without any strong feelings.

Reply:
I have not any feelings about this, just so it is right :) maybe keep current solution and create separate PR for that

Reply:
removed in 8a306814b and I create separate PR